units/host_drawdown.go

```diff
@@ -91,64 +90,45 @@ func (j *hostDrawdownJob) Run(ctx context.Context) {
 		if drawdownTarget <= 0 {
 			break
 		}
-		err = j.checkAndTerminateHost(ctx, &idleHost, &drawdownTarget)
-		if err != nil {
-			grip.Error(message.WrapError(err, message.Fields{
-				"id":             j.ID(),
-				"distro_id":      j.DrawdownInfo.DistroID,
-				"idle_host_list": idleHosts,
-				"message":        "terminate host drawdown error",
-			}))
-		}
+		err = j.checkAndDecommission(ctx, &idleHost, &drawdownTarget)
+		grip.Error(message.WrapError(err, message.Fields{
+			"id":        j.ID(),
+			"distro_id": j.DrawdownInfo.DistroID,
+			"host":      idleHost.Id,
+			"message":   "decommission host drawdown error",
+		}))
 	}
 
 	grip.Info(message.Fields{
-		"id":                   j.ID(),
-		"job_type":             hostDrawdownJobName,
-		"distro_id":            j.DrawdownInfo.DistroID,
-		"new_cap_target":       j.DrawdownInfo.NewCapTarget,
-		"existing_host_count":  existingHostCount,
-		"num_idle_hosts":       len(idleHosts),
-		"num_terminated_hosts": j.Terminated,
-		"terminated_hosts":     j.TerminatedHosts,
+		"id":                       j.ID(),
+		"job_type":                 hostDrawdownJobName,
+		"distro_id":                j.DrawdownInfo.DistroID,
+		"new_cap_target":           j.DrawdownInfo.NewCapTarget,
+		"existing_host_count":      existingHostCount,
+		"num_idle_hosts":           len(idleHosts),
+		"num_decommissioned_hosts": j.Decommissioned,
+		"decommissioned_hosts":     j.DecommissionedHosts,
 	})
 }
 
-func (j *hostDrawdownJob) checkAndTerminateHost(ctx context.Context, h *host.Host, drawdownTarget *int) error {
-
+func (j *hostDrawdownJob) checkAndDecommission(ctx context.Context, h *host.Host, drawdownTarget *int) error {
 	exitEarly, err := checkTerminationExemptions(ctx, h, j.env, j.Type().Name, j.ID())
 	if exitEarly || err != nil {
 		return err
 	}
 
-	// don't drawdown hosts that are currently in the middle of tearing down a task group
+	// Don't drawdown hosts that are currently in the middle of tearing down a task group.
```

Comment:
I might just be reading this wrong, but is the logic in TeardownTimeExceededMax flipped? Shouldn't it return true if we _have_ exceeded max?